app/src/common/utils/validation/validate.js

```diff
@@ -94,6 +94,8 @@ export const attributeKey = (value) =>
 export const uniqueAttributeKey = (attributes) => (value) =>
   attributes.filter((attribute) => attribute === value).length <= 1;
 export const attributeValue = composeValidators([isNotEmpty, maxLength(512)]);
+export const attributesFilterValue = (value) =>
+  !value || composeValidators([isNotOnlySpaces, maxLength(512)])(value);
```

Comment:
Do we need to check `!value` here?
I see the `isNotOnlySpaces` and it has check for this.

![image](https://user-images.githubusercontent.com/31861805/225836223-27075a87-49f7-47be-9ddd-e0ef07e02e2f.png)

Reply:
Yes, we definitely need it here.
The reason for it is the following: when we clear the value of attr, it sets to `null`, and that breaks maxValue validator.
MaxValue validator does not check the value for `null` it only checks for `undefined`.
We can change the code of maxValue validator.